docs/content/about/tech-specs.md

```diff
@@ -16,63 +16,28 @@ The following tables list the software versions NGINX Service Mesh supports and
 ### Supported Versions
 
 {{% table %}}
-| NGINX Service Mesh | Kubernetes  | NGINX Plus Ingress Controller (ingress) | NGINX Plus Ingress Controller (egress) | OpenShift  | Helm     | Rancher  |
-|--------------------|-------------|-----------------------------------------|----------------------------------------|------------|----------|----------|
-| v1.7.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.6, 2.7 |
-| v1.6.0             | 1.22 - 1.24 | 1.12, 2.2, 2.3                          | 2.2, 2.3                               | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.5.0             | 1.22 - 1.24 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.9 - 4.10 | 3.2+     | 2.5, 2.6 |
-| v1.4.1             | 1.18 - 1.22 | 1.12, 2.1, 2.2                          | 2.2                                    | 4.8        | 3.2+     | 2.5, 2.6 |
-| v1.4.0             | 1.18 - 1.22 | 1.12, 2.0, 2.1                          | 1.12, 2.0, 2.1                         | 4.8        | 3.2+     | 2.5, 2.6 |  
-| v1.3.1             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.3.0             | 1.18 - 1.22 | 1.12, 2.0                               | 1.12, 2.0                              | 4.8        | 3.2+     | 2.5, 2.6 |     
-| v1.2.1             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.2.0             | 1.16 - 1.21 | 1.12.0                                  | 1.12.0                                 | 4.8        | 3.5, 3.6 | 2.6      |
-| v1.1.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.1             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-| v1.0.0             | 1.16 - 1.20 | 1.11                                    | 1.11                                   | --         | --       | --       |
-
+| NGINX Service Mesh | Kubernetes | OpenShift | NGINX Ingress Controller | NGINX Plus Ingress Controller | Helm     | Rancher  |
+|--------------------|------------|-----------|--------------------------|-------------------------------|----------|----------|
+| v1.7.0 - latest    | 1.22+      | 4.9+      | 3.0.1+                   | 2.2+                          | 3.2+     | 2.5+     |
```

Comment:
I'm wondering now for consistency with the other table if we just do 1.7.0+ (or the other table add `- latest`)

Reply:
Good call. Changed it to `v1.7.0+` to be consistent with other values.